src/View/layout.html.twig

```diff
@@ -5,17 +5,24 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>{% block title %}{% endblock %}</title>
+    <title>{% block title %}Clevertricks{% endblock %}</title>
 
     <link rel="stylesheet" href="/assets/css/style.css">
     {% block stylesheet %}{% endblock %}
     <link rel="icon" href="/assets/images/favicon.png">
 </head>
 
 <body>
+    <header>
+        {% block nav %}
+        {% include 'Home/nav.html.twig' %}
+        {% endblock %}
```

Comment:
You'd better have the navbar HTML part here if you want your navbar to be on every page.